src/swiss_dict.jl

```diff
@@ -0,0 +1,559 @@
+import Base: setindex!, sizehint!, empty!, isempty, length, copy, empty,
+             getindex, getkey, haskey, iterate, @propagate_inbounds, ValueIterator,
+             pop!, delete!, get, get!, isbitstype, in, hashindex, isbitsunion,
+             isiterable, dict_with_eltype, KeySet, Callable, _tablesz, filter!
```

Comment:
Do we always do this per file?
I thought we just did this at the top level file

Reply:
it's there in `robin_dict.jl`. Nice suggestion, let me get to it and try to get it out from there as well. I'll remove this from here as well.